examples/conf-quickstart/broker/runtime.properties

```diff
@@ -0,0 +1,16 @@
+druid.service=druid/broker
+druid.port=8082
+
+# HTTP server threads
+druid.broker.http.numConnections=5
+druid.server.http.numThreads=8
+
+# Processing threads and buffers
+druid.processing.buffer.sizeBytes=256000000
+druid.processing.numThreads=2
+
+# Query cache (we use a small local cache)
+druid.broker.cache.useCache=true
+druid.broker.cache.populateCache=true
+druid.cache.type=local
+druid.cache.sizeInBytes=10000000
```

Comment:
The main reason this is recommended in the getting started is because caching on broker is faster for smaller queries and smaller clusters. This is important when people are benchmarking Druid in POCs.